notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>We are calling it Tabular Workflow for Forecasting, right? https://cloud.devsite.corp.google.com/vertex-ai/docs/tabular-data/tabular-workflows/forecasting?db=reznitskii</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Sure, I can update it to <code>Tabular Workflow for Forecasting</code>, but I will keep the name to match the tabular workflow for tables.  </p>